Jellyfin.Server/SocketSharp/RequestMono.cs

```diff
@@ -547,15 +557,23 @@ public override string ToString()
                 }
             }
 
+            private const byte LF = (byte)'\n';
+
+            private const byte CR = (byte)'\r';
```

Comment:
Unrelated but, what a mess. The file is even called *Mono.cs. Maybe it's not used?

These values do not deserve constants, since they are by definition constant so they might as well be literals.

Reply:
It certainly used, but these changes are out of the scope of this PR.